symengine/tests/eval/test_lambda_double.cpp

```diff
@@ -204,8 +204,10 @@ TEST_CASE("Check llvm and lambda are equal", "[llvm_double]")
         r = mul(add(pow(integer(2), E), add(r, pow(x, pow(E, cos(x))))), r);
     }
 
+    // r = add(add(x, y), pow(add(x, y), integer(2)));
+
     LLVMDoubleVisitor v;
-    v.init({x, y, z}, *r);
+    v.init({x, y, z}, *r, true);
```

Comment:
The test here is that it still gives the right answer, even if it's using cse, correct?

Reply:
Yes. There's no other way to check given the current API.

Reply:
Ok. That's fine.